src/docs/releases/1.9.0.md

```diff
@@ -353,3 +353,8 @@ Enhanced functionality has been implemented, giving developers the ability to co
  ```csharp
  services.Configure<EmailConfirmationTokenProviderOptions>(options => options.TokenLifespan = TimeSpan.FromDays(7));
  ```
+
+### SectionDisplayDriver
+
+SectionDisplayDriver now uses the appropriate UpdateEditorContext on it's Update methods.
+Every driver that inherits from it will require to update their contexts.
```

Comment:
```suggestion
### `SectionDisplayDriver.UpdateAsync()` now receives an `UpdateEditorContext` parameter, not `BuildEditorContext`

`SectionDisplayDriver` now uses the appropriate UpdateEditorContext on its Update methods. Every driver that inherits from it will be required to update their contexts. See [the corresponding pull request](https://github.com/OrchardCMS/OrchardCore/pull/15488) for context and for examples of the changes you have to implement in your own modules.
```